package.json

```diff
@@ -55,27 +55,26 @@
     "cssnano": "3.10.0",
     "fs-extra": "1.0.0",
     "git-rev-sync": "1.4.0",
-    "grunt": "0.4.5",
-    "grunt-cli": "0.1.13",
-    "grunt-contrib-connect": "0.11.2",
-    "grunt-contrib-copy": "0.8.0",
+    "grunt": "^1.0.3",
```

Comment:
We are moving from specific versions to being more flexible by allowing the most recent major version now ? I guess its ok for grunt, but I think we are not consistent about this pattern.

Reply:
yeah, we're not quite consistent. I'm being more sloppy since we have the lock file.